src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions/src/RazorPageDocumentClassifierPass.cs

```diff
@@ -11,9 +11,16 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
 {
     public class RazorPageDocumentClassifierPass : DocumentClassifierPassBase
     {
+        private bool _useConsolidatedMvcViews = false;
+
         public static readonly string RazorPageDocumentKind = "mvc.1.0.razor-page";
         public static readonly string RouteTemplateKey = "RouteTemplate";
 
+        public RazorPageDocumentClassifierPass(bool useConsolidatedMvcViews = false)
```

Comment:
Same deal. I think we might need a better name than `useConsolidated...` but we can figure that out in API review.